src/harness/testcases/WINNF_FT_S_GRA_testcase.py

```diff
@@ -276,6 +276,232 @@ def test_WINNF_FT_S_GRA_4(self):
     self.assertFalse('grantId' in response)
     self.assertEqual(response['response']['responseCode'], 103)
 
+  def generate_GRA_5_default_config(self, filename):
+    """Generates the WinnForum configuration for GRA_5"""
+    # Create the actual config for GRA_5
+
+    # Load device_c1.
+    device_c1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+
+    # Load grant request.
+    grant_g1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+
+    # Load device_c2.
+    device_c2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Creating conditionals for Cat B devices.
+    self.assertEqual(device_c2['cbsdCategory'], 'B')
+    conditional_parameters = {
+        'cbsdCategory': device_c2['cbsdCategory'],
+        'fccId': device_c2['fccId'],
+        'cbsdSerialNumber': device_c2['cbsdSerialNumber'],
+        'airInterface': device_c2['airInterface'],
+        'installationParam': device_c2['installationParam'],
+        'measCapability': device_c2['measCapability']
+    }
+    del device_c2['cbsdCategory']
+    del device_c2['airInterface']
+    del device_c2['installationParam']
+
+    # Load grant request
+    grant_g2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+
+    conditionals = {
+        'registrationData': [conditional_parameters]
+    }
+
+    cbsd_record = [device_c1]
+    grant_record = [grant_g1]
+
+    sas_harness_config = {
+        'sasTestHarnessName': 'SAS-Test-Harness-1',
+        'hostName': 'localhost',
+        'port': 9001,
+        'serverCert': "certs/server.cert",
```

Comment:
Use os.path.join() instead so that this will work on other OSes.

Reply:
changed as suggested